lib/keyboard_layouts.dart

```diff
@@ -115,6 +115,36 @@ const graphite = KeyboardLayout(
   ],
 );
 
+const galliumV2 = KeyboardLayout(
+  name: 'Gallium V2',
+  keys: [
+    ['B', 'L', 'D', 'W', 'Z', "'", 'F', 'O', 'U', 'J', ';', '='],
+    ['N', 'R', 'T', 'S', 'G', 'Y', 'H', 'A', 'E', 'I', ','],
+    ['Q', 'X', 'M', 'C', 'V', 'K', 'P', '.', '-', '/'],
+    [' '],
+  ],
+);
+
+const sturdy = KeyboardLayout(
+  name: 'Sturdy',
+  keys: [
+    ['V', 'M', 'L', 'C', 'P', "X", 'F', 'O', 'U', '-', '[', ']'],
+    ['S', 'T', 'R', 'D', 'Y', '.', 'N', 'A', 'E', 'I', '/'],
+    ['Z', 'K', 'Q', 'G', 'W', 'B', 'H', "'", ';', ','],
```

Comment:
ZKQGW ordering is for split keyboards. Change into Sturdy ANSI with Angle Mod, i.e., KQGWZ. (This is just based on MonkeyType as I'm not familiar with Sturdy layout.